components/colorwheel/index.css

```diff
@@ -56,35 +94,68 @@
   }
 }
 
-.spectrum-ColorWheel-gradient {
-  position: relative;
-  z-index: 0;
-  width: var(--spectrum-colorwheel-width);
-  height: var(--spectrum-colorwheel-height);
-  border-radius: 100%;
-  box-sizing: border-box;
-
-  border-width: var(--spectrum-colorwheel-border-size);
-  border-style: solid;
+.spectrum-ColorWheel-internal{
+  display: flex;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+  width: var(--spectrum-colorwheel-colorarea-container-size);
+  height: var(--spectrum-colorwheel-colorarea-container-size);
+  margin: auto;
 }
 
-.spectrum-ColorWheel-wheel {
-  position: relative;
-  z-index: 1;
-}
-
-.spectrum-ColorWheel-outerCircle,
-.spectrum-ColorWheel-innerCircle {
-  fill: transparent;
-
-  stroke-width: var(--spectrum-colorwheel-border-size);
+.spectrum-ColorWheel-colorarea-container{
+  height: auto;
+  width: 100%;
+  margin: var(--mod-colorwheel-colorarea-margin, var(--spectrum-colorwheel-colorarea-margin));
 }
 
 .spectrum-ColorWheel-slider {
   @extend %spectrum-ColorControl-hiddenField;
 }
 
 .spectrum-ColorWheel-handle {
+  transform: translate(var(--spectrum-colorwheel-colorhandle-position), 0);
   left: 50%;
   top: 50%;
 }
+
+/* clip path set border-width wider than than spectrum-colorwheel-wheel to create appreance of a border*/
+.spectrum-ColorWheel-border{
+  position: relative;
+  background-color: var(--mod-colorwheel-border-color, var(--spectrum-colorwheel-border-color));
+  width: var(--mod-colorwheel-width, var(--spectrum-colorwheel-width));
+  height: var(--mod-colorwheel-height, var(--spectrum-colorwheel-height));
+  clip-path: path(evenodd, var(--spectrum-colorwheel-path-borders));
+
+  &.is-disabled{
+    background-color: var(--highcontrast-colorwheel-border-color-disabled, var(--spectrum-colorwheel-border-color));
+  }
+}
+
+/* hack: make sure we don't complain about these variables being unused*/
+.🤫 {
+  width: var(--track-width);
+  border-width: var(--border-width);
+}
+
+/* --track-width and --border-width to be used with JS in calculating the clip-path paths and colorarea-container-size */
+.spectrum-ColorWheel-wheel{
+  position: absolute;
+  background: conic-gradient(from 90deg, red, rgb(255, 128, 0), rgb(255, 255, 0), rgb(128, 255, 0), rgb(0, 255, 0), rgb(0, 255, 128), rgb(0, 255, 255), rgb(0, 128, 255), rgb(0, 0, 255), rgb(128, 0, 255), rgb(255, 0, 255), rgb(255, 0, 128), red);
+  top: 1px;
+  left: 1px;
+  width: 100%;
+  height: 100%;
```

Comment:
Similar to my comment above about logical properties. I think we can convert these to [logical properties](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties).